src/overlays/actors/ovl_En_Syateki_Man/z_en_syateki_man.c

```diff
@@ -1261,36 +1261,36 @@ void EnSyatekiMan_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dL
     EnSyatekiMan* this = THIS;
     Vec3f sp18 = { 1600.0f, 0.0f, 0.0f };
 
-    if (limbIndex == 15) {
+    if (limbIndex == SHOOTING_GALLERY_MAN_LIMB_HEAD) {
         Matrix_MultiplyVector3fByState(&sp18, &this->actor.focus.pos);
     }
 }
 
 void EnSyatekiMan_Draw(Actor* thisx, GlobalContext* globalCtx) {
-    static TexturePtr D_809C94B8[] = {
-        object_shn_Tex_005AC8,
-        object_shn_Tex_0062C8,
-        object_shn_Tex_0062C8,
+    static TexturePtr sEyeTextures[] = {
+        gSwampShootingGalleryManEyeOpenTex,
+        gSwampShootingGalleryManEyeHalfTex,
+        gSwampShootingGalleryManEyeHalfTex,
     };
     EnSyatekiMan* this = THIS;
     s32 pad;
 
     if (globalCtx->sceneNum == SCENE_SYATEKI_MIZU) {
-        D_809C94B8[0] = object_shn_Tex_00FB90;
-        D_809C94B8[1] = object_shn_Tex_010390;
-        D_809C94B8[2] = object_shn_Tex_010390;
+        sEyeTextures[0] = gTownShootingGalleryManEyeOpenTex;
+        sEyeTextures[1] = gTownShootingGalleryManEyeClosedTex;
+        sEyeTextures[2] = gTownShootingGalleryManEyeClosedTex;
     } else {
-        D_809C94B8[0] = object_shn_Tex_005AC8;
-        D_809C94B8[1] = object_shn_Tex_0062C8;
-        D_809C94B8[2] = object_shn_Tex_0062C8;
+        sEyeTextures[0] = gSwampShootingGalleryManEyeOpenTex;
+        sEyeTextures[1] = gSwampShootingGalleryManEyeHalfTex;
+        sEyeTextures[2] = gSwampShootingGalleryManEyeHalfTex;
     }
 
     OPEN_DISPS(globalCtx->state.gfxCtx);
 
     func_8012C5B0(globalCtx->state.gfxCtx);
 
-    gSPSegment(POLY_OPA_DISP++, 0x08, Lib_SegmentedToVirtual(D_809C94B8[this->unk_264]));
-    gSPSegment(POLY_OPA_DISP++, 0x09, Lib_SegmentedToVirtual(D_809C94B8[this->unk_264]));
+    gSPSegment(POLY_OPA_DISP++, 0x08, Lib_SegmentedToVirtual(sEyeTextures[this->eyeIndex]));
+    gSPSegment(POLY_OPA_DISP++, 0x09, Lib_SegmentedToVirtual(sEyeTextures[this->eyeIndex]));
```

Comment:
Wait, what? Is this like Great Fairies where one set is just not used? And it overwrites them with themselves?

Reply:
Both sets are used, but it starts with the Swamp set, then...overwrites it with the Swamp set for the Swamp Shooting Gallery Man anyways. It's a pointless operation. The *real* weird thing is that the Swamp Shooting Gallery Man's eyes never fully close; he just uses the half texture twice.

Reply:
I was looking at the two segments in particular here: does the head use both segments?

Reply:
Oh, I see what you mean. Yeah, the head only uses segment 8. The segment 9 thing here is pointless.

Reply:
Yeah, so just like Great Fairy. Shame.